pipeline/load/daily_bus_ridership.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS daily_bus_ridership;
+CREATE TABLE daily_bus_ridership(
+    route INTEGER,
+    date DATE,
+    daytype TEXT,
+    rides INTEGER
```

Comment:
let's add a week column to our table to incorporate the week that we added on the transform step
it would look like this:
```
rides INTEGER,
week TEXT
```